src/include/deepstate/DeepState.h

```diff
@@ -753,39 +759,100 @@ static int DeepState_RunTestNoFork(struct DeepState_TestInfo *test) {
     DeepState_LogFormat(DeepState_LogTrace, "Passed: %s", test->test_name);
     if (HAS_FLAG_output_test_dir) {
       if (!FLAGS_fuzz || FLAGS_fuzz_save_passing) {
-	DeepState_SavePassingTest();
+        DeepState_SavePassingTest();
       }
     }
     return(DeepState_TestRunPass);
   }
 }
 
+/* Run a test case inside a new Windows process */
+static int DeepState_RunTestWin(struct DeepState_TestInfo *test){
+
+  #if defined(_WIN32) || defined(_MSC_VER)
+
+    PROCESS_INFORMATION pi;
+    STARTUPINFO si;
+    DWORD exit_code = 0;
```

Comment:
Good point; by default, the exit code should be interpreted as `DeepState_TestRunPass` which indicates that the test was successful: fixed this with commit . 

However, I don't get the `int *failed` argument's logic. Do you intend to include it in case the test exits with a code other than `0` without necessarily meaning that the test failed?

Reply:
@FabrizioSandri it wasn't clear to me that failure was being sufficiently distinguished from other return values. E.g. normally a successful exit code on a UNIX process is `0`. If failure needs to be distinguished and can already be distinguished with the existing return values then I am fine without additional arguments.

Reply:
Yes, a successful run can already be recognized from a failure. DeepState calls `DeepState_RunSingle()` after a new process is launched, which then calls `DeepState_RunTest()`. The latter terminates with `DeepState_TestRunPass` if the tested function terminates successfully regardless of the exit code, i.e. if the test returns correctly without encountering an exception. This means that the second line of the following snippet of code is reached, and the test is deemed successful regardless of the return value of `test->test_func()`. 

```c++
/* Snippet taken from DeepState_RunTest */
test->test_func(); 
exit(DeepState_TestRunPass);
```

On the other hand `DeepState_RunTest()` is guaranteed to exit with an exit code other than `0` in the event that the test fails, and this situation is thus caught in the condition that I inserted in commit 94f6bb1.